libmachine/provision/custom_script.go

```diff
@@ -27,7 +27,10 @@ func WithCustomScript(provisioner Provisioner, customScriptPath string) error {
 		return fmt.Errorf("unable to read file %s: %v", customScriptPath, err)
 	}
 
-	if output, err := provisioner.SSHCommand(fmt.Sprintf("echo \"%v\" | sh -", string(customScriptContents))); err != nil {
+	if output, err := provisioner.SSHCommand(fmt.Sprintf("cat <<'OEOF' >install_script.sh\n%s\nOEOF", string(customScriptContents))); err != nil {
```

Comment:
can we maybe put the script in /tmp/install_script.sh and call if from there and `rm -f` it afterwardS?

Reply:
Moved the script to `/tmp/install_script.sh`.